scripts/CheckSecurityConfigs.s.sol

```diff
@@ -50,130 +50,301 @@ contract CheckSecurityConfigs is Script {
+    function checkSystemConfigProxy(
+        ProtocolAddresses memory addresses
+    ) internal {
+        console2.log(
+            "Checking SystemConfigProxy %s",
+            addresses.SystemConfigProxy
+        );
         isAdminOf(addresses.ProxyAdmin, addresses.SystemConfigProxy);
         isOwnerOf(addresses.SystemConfigOwner, addresses.SystemConfigProxy);
     }
 
     function checkProxyAdmin(ProtocolAddresses memory addresses) internal {
         console2.log("Checking ProxyAdmin %s", addresses.ProxyAdmin);
         isOwnerOf(addresses.ProxyAdminOwner, addresses.ProxyAdmin);
-        checkAddressIsExpected(addresses.AddressManager, addresses.ProxyAdmin, "addressManager()");
+        checkAddressIsExpected(
+            addresses.AddressManager,
+            addresses.ProxyAdmin,
+            "addressManager()"
+        );
     }
 
-    function getMappingValue(address targetContract, uint256 mapSlot, address key) public view returns (uint256) {
-        bytes32 slotValue = vm.load(targetContract, keccak256(abi.encode(key, mapSlot)));
+    function getMappingValue(
+        address targetContract,
+        uint256 mapSlot,
+        address key
+    ) public view returns (uint256) {
+        bytes32 slotValue = vm.load(
+            targetContract,
+            keccak256(abi.encode(key, mapSlot))
+        );
         return uint256(slotValue);
     }
 
-    function isAdminOf(address expectedOwner, address ownableContract) internal {
+    function isAdminOf(
+        address expectedOwner,
+        address ownableContract
+    ) internal {
         checkAddressIsExpected(expectedOwner, ownableContract, "admin()");
     }
 
-    function isOwnerOf(address expectedOwner, address ownableContract) internal {
+    function isOwnerOf(
+        address expectedOwner,
+        address ownableContract
+    ) internal {
         checkAddressIsExpected(expectedOwner, ownableContract, "owner()");
     }
 
-    function checkAddressIsExpected(address expectedAddr, address contractAddr, string memory signature) internal {
+    function checkAddressIsExpected(
+        address expectedAddr,
+        address contractAddr,
+        string memory signature
+    ) internal {
         address actual = getAddressFromCall(contractAddr, signature);
         if (expectedAddr != actual) {
-            console2.log("  !! Error: %s != %s.%s, ", expectedAddr, contractAddr, signature);
+            console2.log(
+                "  !! Error: %s != %s.%s, ",
+                expectedAddr,
+                contractAddr,
+                signature
+            );
             console2.log("           which is %s", actual);
             hasErrors = true;
         } else {
-            console2.log("  -- Success: %s == %s.%s.", expectedAddr, contractAddr, signature);
+            console2.log(
+                "  -- Success: %s == %s.%s.",
+                expectedAddr,
+                contractAddr,
+                signature
+            );
```

Comment:
The repo is not currently opinionated about formatting, and my editor automatically formatted this file, resulting in a noisy diff.

Reply:
Correction, we can run `forge fmt`. Need to update my editor config.